manifests/enterprise/dashboard/package.pp

```diff
@@ -0,0 +1,33 @@
+class sensu::enterprise::dashboard::package {
+
+  if $caller_module_name != $module_name {
+    fail("Use of private class ${name} by ${caller_module_name}")
+  }
+
+  if $::sensu::install_repo and $::sensu::enterprise_dashboard {
+    $ensure = 'present'
+
+    $se_user = $sensu::enterprise_user
+    $se_pass = $sensu::enterprise_pass
+
+    $url = "http://${se_user}:${se_pass}@enterprise.sensuapp.com/yum/\$basearch/"
+
+    yumrepo { 'sensu-enterprise-dashboard':
```

Comment:
I think this should be moved to sensu::repos::yum so it honors the $install_repo flag and it seems odd to install a repo somewhere else since that exists.